examples/flags/go.mod

```
@@ -0,0 +1,203 @@
+module flags
+
+go 1.21.1
+
+toolchain go1.21.3
+
+require (
+	github.com/Code-Hex/Neo-cowsay/v2 v2.0.4
+	github.com/hashicorp/go-plugin v1.5.2
+	github.com/ignite/cli/v28 v28.0.0
```

Ignite dependency could be updated to the latest so it includes support for the new `.ign` app extension:

```suggestion
	github.com/ignite/cli/v28 v28.1.1
```